src/test/java/de/mediathekview/mserver/crawler/ndr/tasks/NdrSendungsfolgeDetailsTaskTest.java

```diff
@@ -0,0 +1,126 @@
+            {
+                "https://www.ndr.de/fernsehen/sendungen/sass-so-isst-der-norden/Sass-So-isst-der-Norden,sendung563148.html",
+                "/ndr/ndr_film_detail1.html",
+                "/fernsehen/sendungen/sass-so-isst-der-norden/sass404-ardjson_image-5a3e2524-70e1-45f1-96da-27bf9d5c8137.json",
+                "/ndr/ndr_film_detail1.json",
+                "Sass: So isst der Norden",
+                "Deftige Eintopfgerichte aus Bremen",
+                "Rainer Sass macht mit seiner mobilen Küche Station auf dem Bremer Domshof. Mit seinen Kochpartnern vom dortigen Markt bereitet er Deftiges aus frischem Gemüse zu.",
+                LocalDateTime.of(2018, 3, 18, 16, 30, 0),
+                Duration.ofMinutes(30).plusSeconds(6),
+                "https://mediandr-a.akamaihd.net/progressive/2016/1009/TV-20161009-1036-1800.hi.mp4",
+                "https://mediandr-a.akamaihd.net/progressive/2016/1009/TV-20161009-1036-1800.hq.mp4",
+                "https://mediandr-a.akamaihd.net/progressive/2016/1009/TV-20161009-1036-1800.hd.mp4",
+                "",
+                GeoLocations.GEO_NONE
+            },
+            {
+                "https://www.ndr.de/fernsehen/Folge-2881-Der-langersehnte-Antrag,sturmderliebe1816.html",
+                "/ndr/ndr_film_detail2.html",
+                "/fernsehen/sturmderliebe1816-ardjson_image-a5409105-e38b-4847-ba32-fc2c337d7515.json",
+                "/ndr/ndr_film_detail2.json",
+                "Sturm der Liebe",
+                "Folge 2881: Der langersehnte Antrag",
+                "André fasst sich ein Herz und macht Melli den geplanten Antrag. Währenddessen ist Romy enttäuscht, dass Paul lediglich eine gute Freundin in ihr sieht.",
+                LocalDateTime.of(2018, 3, 15, 8, 10, 0),
+                Duration.ofMinutes(48).plusSeconds(34),
+                "https://mediandr-a.akamaihd.net/progressive_geo/2018/0315/TV-20180315-0915-0800.hi.mp4",
+                "https://mediandr-a.akamaihd.net/progressive_geo/2018/0315/TV-20180315-0915-0800.hq.mp4",
+                "https://mediandr-a.akamaihd.net/progressive_geo/2018/0315/TV-20180315-0915-0800.hd.mp4",
+                "https://www.ndr.de/media/ut65560.html",
+                GeoLocations.GEO_DE
+            }
+        });
+  }
+
+  private final String requestUrl;
+  private final String htmlPage;
+  private final String jsonUrl;
+  private final String jsonFile;
+  private final String expectedTopic;
+  private final String expectedTitle;
+  private final String expectedDescription;
+  private final LocalDateTime expectedTime;
+  private final Duration expectedDuration;
+  private final String expectedUrlSmall;
+  private final String expectedUrlNormal;
+  private final String expectedUrlHd;
+  private final String expectedSubtitle;
+  private final GeoLocations expectedGeo;
+
+  public NdrSendungsfolgeDetailsTaskTest(final String aRequestUrl, final String aHtmlPage, final String aJsonUrl, final String aJsonFile,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 14 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)